Library/ViewModels/BackingViewModel.swift

```diff
@@ -256,8 +258,10 @@ public final class BackingViewModel: BackingViewModelType, BackingViewModelInput
       projectAndBackingAndBackerIsCurrentUser.mapConst(1.0)
     )
 
-    self.rootStackViewAxis = projectAndBackingAndBackerIsCurrentUser
-      .map { _ in AppEnvironment.current.language == .en ? .horizontal : .vertical }
+    self.actionsStackViewAxis = Signal.merge(
+      self.viewDidLoadProperty.signal,
+      self.viewWillTransitionProperty.signal
+    ).map { _ in AppEnvironment.current.device.orientation.isPortrait ? .vertical : .horizontal }
```

Comment:
I think instead of inspecting the orientation we actually want to check the `UITraitCollection`'s `sizeClass`.

So this would be `horizontalSizeClass == .regular` and for this we would have an input for `traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?)`.